src/test/java/com/znsio/teswiz/steps/VodQASteps.java

```diff
@@ -114,4 +114,9 @@ public void iAmAbleToViewMessage(String displayedMessage) {
     public void iShouldBeAbleToDoubleTapOnAnElement() {
         new VodqaBL().doubleTapOnAnElement();
     }
+
+    @Then("I should be able to pinch and zoom on an element")
+    public void iShouldBeAbleToPinchAndZoomOnAnElement() {
+        new VodqaBL().pinchAndZoomOnAndElement();
```

Comment:
Here by reading this it seems only zoom in functionality is present when running the method, but later you have integrated both zoom in and zoom out.
You can either rename this method and Then feature line accordingly or make a separate And after the Then line

Reply:
Implemented the same